deployments/editor/libero-editor--prod.yaml

```diff
@@ -30,6 +30,8 @@ spec:
         tag: 0.0.3
 
     database:
+      endpoint: 'documentdbcluster-2287lljntodp.cluster-cxyopn44uqbl.us-east-1.docdb.amazonaws.com:27017'
+      query: 'ssl=true&replicaSet=rs0&readPreference=secondaryPreferred'
```

Comment:
Something that we might want to followup on in another issue, I recall from the code that there is a separate flag for determining if we should use ssl or not, as such the `ssl=true` arg should probably be programatically appended in the code rather than set here. Not an issue for our deployment however, so something to worry about further down the line.

Reply:
Will follo up post-MVP https://github.com/libero/editor-article-store/pull/41#discussion_r561707610